tests/scripts/pulp_gem/test_remote.sh

```diff
@@ -12,9 +12,9 @@ trap cleanup EXIT
 
 expect_succ pulp gem remote list
 
-expect_succ pulp gem remote create --name "cli_test_gem_remote" --url "$GEM_REMOTE_URL"
+expect_succ pulp gem remote create --name "cli_test_gem_remote" --url "$GEM_REMOTE_URL" --includes '{"panda":null}'
 
-expect_succ pulp gem remote update --remote "cli_test_gem_remote" --policy "on_demand"
+expect_succ pulp gem remote update --remote "cli_test_gem_remote" --policy "on_demand" --prereleases "True"
```

Comment:
We can either set, reset or ignore this boolean flag, right? Would we get this tristate from click if we made it a `"--prereleases/--no-prereleases"` flag?

Reply:
Let me test.

Reply:
Yep, it does work as a tristate when using the flag option.